### PR TITLE
Add documentation for saving attributes

### PR DESCRIPTION
**Pull request by Lloople**
Adds documentation requested in #62

### CODE REVIEW

File: docs/usage/forms.md [hunk 1]
```diff
@@ -15,8 +15,31 @@ $post->fill([
...
 
-```markup
+```html
 <input type="text" name="en[title]" />
 <input type="text" name="de[title]" />
 ```
 
+# Saving the request directly
+
+If you want to use `$post->update(request()->all())` you can structure your input in semicolon notation.
```
Comment (Gummibeer) [inline docs/usage/forms.md]: It's a colon not a semicolon. 😉

File: docs/usage/forms.md [hunk 2]
```diff
@@ -15,8 +15,31 @@ $post->fill([
 
 To achieve this structure in your form - to prevent manipulating the form data just to save them. You can use the input name array `[]` [syntax](https://www.php.net/manual/en/faq.html.php#faq.html.arrays).
 
-```markup
+```html
 <input type="text" name="en[title]" />
 <input type="text" name="de[title]" />
 ```
 
+# Saving the request directly
```
Comment (Gummibeer) [inline docs/usage/forms.md]: I'm not really happy with this naming. Filling works also with `en[name]`. This naming indicates that this is the preferred approach. For me it isn't - that's personal opinion.^^
At all I would like to keep the docs unopinionated. Could you find a naming that fits better?

Comment (Lloople) [reply docs/usage/forms.md]: I'll try to structure the section by `request`, and see what you think about it 👍

File: docs/usage/forms.md [hunk 3]
```diff
@@ -13,10 +13,40 @@ $post->fill([
...
+  $post->update($request->all());
+}
+```
+
+# Updating the default locale
+
+You can update the default locale attributes by accessing the translatable attributes from the model itself.
+
+```php
+$post->title = 'My first post'; // Will update the English translatable model
```
Comment (Gummibeer) [inline docs/usage/forms.md]: A `fill()` line that uses an array with a simple title key would show that it also works with fill. 😊

Comment (Lloople) [reply docs/usage/forms.md]: you mean `$post->fill(['title' => 'My first post']);`? Didn't know that! 🔥

**Pull request merged**